pkg/db/migrations/202401151014_add_server_instances.go

```diff
@@ -9,6 +11,8 @@ import (
 func addServerInstances() *gormigrate.Migration {
 	type ServerInstance struct {
 		Model
+		LastHeartbeat time.Time
+		Ready         bool `gorm:"default:false"`
 	}
 
```

Comment:
This should have been a new migration and not an edit of an existing migration.

@morvencao @skeeey @clyang82 because of this, the migration doesn't get applied to existing deployments as it was already applied